.github/workflows/tf-deploy-basic.yaml

```diff
@@ -1,42 +1,64 @@
 on:
   workflow_call:
     inputs:
-      account_id:
+      artifact_path:
+        description: "The path of the artifacts to download"
         type: string
-        required: true
-      environment:
+        required: false
+        default: ""
+      aws_account_id:
+        description: "The AWS account ID"
         type: string
         required: true
-      feedback:
-        required: false
-        type: boolean
-        default: false
-      region:
+      aws_region:
+        description: "The AWS region"
         required: true
         type: string
-      role:
+      aws_role_name:
+        description: "The name of the role to assume with OIDC"
         type: string
         required: true
         default: cicd-iac
-      state_file:
-        required: false
+      environment:
```

Comment:
Ok but still not convenience that the name of the variable is right in a public repo ? At least we should explain how this variable will be used (for tagging, ...)

Reply:
(about the "environment" variable name)